include/swift/AST/LocalizationLanguages.def

```diff
@@ -14,10 +14,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#define DEFINE_LOCALIZATION_MACROS
-#include "DefineLocalizationMacros.h"
-
+#ifndef LOCALE_NO_UNDEF
```

Comment:
@xedin Are the macros at the beginning of this file necessary, and if they are... why?

Reply:
If I removed them totally, everything will compiler just fine. But I don't know if I'll be shooting myself in the foot.

Reply:
@LucianoPAlmeida WDYT here?

Reply:
Where `LOCALE` is being defined?

Reply:
I mean the `#define LOCALE(Name, Value)` ...

Reply:
in CompilerInvocation https://github.com/apple/swift/blob/fd215c6056021120477884ded3db9167883fed55/lib/Frontend/CompilerInvocation.cpp#L38